spec/models/legislator_spec.rb

```diff
@@ -227,7 +227,11 @@
 
   describe "#serializable_hash" do
     let(:senator) { FactoryGirl.create(:senator) }
-    let(:keys) { ["id", "chamber", "party", "state_rank", "name", "title", "image_url", "display_district", "state_abbrev", "district_code"] }
+    let(:keys) { [
+        "id", "chamber", "party", "state_rank", "name", "title", "image_url",
+        "display_district", "state_abbrev", "district_code", "eligible",
+        "state_name", "with_us"
+      ] }
```

Comment:
Expression at 234, 9 should be on its own line.<br>Indent the right bracket the same as the start of the line where the left bracket is.